x-pack/plugins/alerting/server/authorization/alerting_authorization_kuery.ts

```diff
@@ -35,22 +37,40 @@ const esQueryConfig: EsQueryConfig = {
 
 export function asFiltersByRuleTypeAndConsumer(
   ruleTypes: Set<RegistryAlertTypeWithAuth>,
-  opts: AlertingAuthorizationFilterOpts
+  opts: AlertingAuthorizationFilterOpts,
+  includeSpaceId = false,
+  spaceId?: string | undefined
```

Comment:
Same comment as above: Would it make sense to fold the space id opts into the AlertingAuthorizationFilterOpts?